sound/soc/intel/boards/sof_es8336.c

```diff
@@ -690,13 +766,29 @@ static int sof_es8336_remove(struct platform_device *pdev)
 	return 0;
 }
 
+static const struct platform_device_id board_ids[] = {
+	{
+		.name = "sof_es83x6_hdmi_ssp",
```

Comment:
use a better name that is more representative of your board. this says nothing about hdmi capture and ssp0.

Reply:
We have modified the name sof_es83x6_hdmi_ssp -> adl_es83x6_hdmi_ssp. Due to the character limit for drv_name, we could not include more information like usage of ssp0 and ssp2.